hack/lib/log/stacktrace.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#
+# This library contains an implementation of a stack trace for Bash scripts. 
+# We assume $OS_ROOT is set
+source "${OS_ROOT}/hack/util.sh"
+
+# os::log::stacktrace::install installs the stacktrace as a handler for the ERR signal if one
+# has not already been installed and sets `set -o errtrace` in order to propagate the handler
+#
+# Globals:
+#  None
+# Arguments:
+#  None
+# Returns:
+#  - export OS_USE_STACKTRACE
```

Comment:
Here and elsewhere, you list library-internal state-tracking variables as returns, which seems like it's stretching the definition of a "return".  Does it really make sense to list such things as returns?